[2023-06-27T17:49:29.254+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: load_pageviews.create_dbt_models manual__2023-06-27T17:48:12.724159+00:00 [queued]>
[2023-06-27T17:49:29.261+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: load_pageviews.create_dbt_models manual__2023-06-27T17:48:12.724159+00:00 [queued]>
[2023-06-27T17:49:29.262+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-27T17:49:29.262+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-27T17:49:29.262+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-27T17:49:29.273+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): create_dbt_models> on 2023-06-27 17:48:12.724159+00:00
[2023-06-27T17:49:29.278+0000] {standard_task_runner.py:55} INFO - Started process 1735 to run task
[2023-06-27T17:49:29.281+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'load_pageviews', 'create_dbt_models', 'manual__2023-06-27T17:48:12.724159+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/load_pageviews.py', '--cfg-path', '/tmp/tmpsvmauj1w']
[2023-06-27T17:49:29.281+0000] {standard_task_runner.py:83} INFO - Job 15: Subtask create_dbt_models
[2023-06-27T17:49:29.321+0000] {task_command.py:389} INFO - Running <TaskInstance: load_pageviews.create_dbt_models manual__2023-06-27T17:48:12.724159+00:00 [running]> on host df36c5c3b1c7
[2023-06-27T17:49:29.371+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_pageviews
AIRFLOW_CTX_TASK_ID=create_dbt_models
AIRFLOW_CTX_EXECUTION_DATE=2023-06-27T17:48:12.724159+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-27T17:48:12.724159+00:00
[2023-06-27T17:49:29.372+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T17:49:29.373+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/helper/dbt_pageviews && dbt run']
[2023-06-27T17:49:29.382+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T17:49:29.383+0000] {subprocess.py:93} INFO - /bin/bash: line 1: cd: /opt/***/helper/dbt_pageviews: No such file or directory
[2023-06-27T17:49:29.384+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-06-27T17:49:29.391+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-06-27T17:49:29.393+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=load_pageviews, task_id=create_dbt_models, execution_date=20230627T174812, start_date=20230627T174929, end_date=20230627T174929
[2023-06-27T17:49:29.402+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 15 for task create_dbt_models (Bash command failed. The command returned a non-zero exit code 1.; 1735)
[2023-06-27T17:49:29.412+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2023-06-27T17:49:29.431+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
